test/unit/math/CMakeLists.txt

```diff
@@ -15,8 +15,7 @@ append_recursive_files_add_to_src_group("src/" "src/" "hpp" _FILES_HEADER)
 
 if(ALPAKA_ACC_GPU_CUDA_ENABLE)
     list(REMOVE_ITEM
-            CUDA_NVCC_FLAGS "--ftz=true" "--prec-div=false" "--prec-sqrt=false" "--fmad=true" "--use_fast_math" "-use_fast_math")
-
+            CMAKE_CXX_FLAGS "--ftz=true" "--prec-div=false" "--prec-sqrt=false" "--fmad=true" "--use_fast_math" "-use_fast_math")
```

Comment:
Shouldn't this be `CMAKE_CUDA_FLAGS`?

Reply:
To be honest, I do not know.
CI was happy. If you want, I could try to remove it completely because if it was wrong after my change, it is unnecessary.

Reply:
Yes, that's a good idea. If it doesn't break the CI we should remove this since it seems unnecessary.

Reply:
please use the new compiler option manipulations from #1289 (`alpaka_set_compiler_options`).

I do not understand why do we remove flags here?

Reply:
> I do not understand why do we remove flags here?

Ahh now I get it, it is for the math test only.